tuna/load_job.py

```diff
@@ -80,11 +76,11 @@ def parse_args():
                              help='add jobs with only these solvers '\
                                '(can be a comma separated list)')
   parser.add_argument(
-      '-o',
-      '--only_applicable',
-      dest='only_app',
+      '-d',
+      '--only_dynamic',
+      dest='only_dynamic',
```

Comment:
The --only_applicable option is no longer available when loading job? It will be either load all configs or load only dynamic?